crates/cibse_guide_e/src/chapter_6/appendix/c_windows_in_more_than_one_wall.rs

```diff
@@ -0,0 +1,72 @@
+pub use super::b_more_than_one_window;
+
+pub fn total_area_of_openings_per_wall(dimensions_of_openings_wall1: Vec<(f64, f64)>) -> f64 {
+    let vector_of_areas_wall1 =
+        b_more_than_one_window::areas_of_openings(dimensions_of_openings_wall1);
+    return b_more_than_one_window::total_area_of_openings(vector_of_areas_wall1);
+}
+
+pub fn total_area_of_openings_per_wall_equation(
+    a_o: String,
+    areas_of_openings: Vec<String>,
+) -> String {
+    return b_more_than_one_window::total_area_of_openings_equation(a_o, areas_of_openings);
+}
+
+pub fn total_area_of_openings(areas_of_openings: Vec<f64>) -> f64 {
+    return b_more_than_one_window::total_area_of_openings(areas_of_openings);
+}
```

Comment:
I like the idea of wrapping the common equation here, but I think that we should do it in both modules. i.e. I think this should look like this:
```suggestion
pub fn total_area_of_openings(areas_of_openings: Vec<f64>) -> f64 {
    return common::total_area_of_openings(areas_of_openings);
}
```

Reply:
Addressed.